core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/StopStartDownscaleDecommissionViaCMHandler.java

```diff
@@ -173,6 +177,19 @@ protected Selectable doAccept(HandlerEvent<StopStartDownscaleDecommissionViaCMRe
         }
     }
 
+    private void excludeAndLogDisconnectedNMsForDownscale(Stack stack, Map<String, InstanceMetadataView> hostsToRemove,
+            StopStartDownscaleDecommissionViaCMRequest request) {
+        ClusterHealthService clusterHealthService = clusterApiConnectors.getConnector(stack).clusterHealthService();
+        Set<String> disconnectedNMHostNames = clusterHealthService.getDisconnectedNodeManagers();
+        if (!disconnectedNMHostNames.isEmpty()) {
+            LOGGER.info("Found {} disconnected NodeManagers: {}, for hostgroup {}. Excluding them from de-commission list", disconnectedNMHostNames.size(),
+                    disconnectedNMHostNames, request.getHostGroupName());
+            flowMessageService.fireEventAndLog(stack.getId(), UPDATE_IN_PROGRESS.name(), CLUSTER_SCALING_STOPSTART_DOWNSCALE_EXCLUDE_LOST_NODES,
+                    String.valueOf(disconnectedNMHostNames.size()), String.join(", ", disconnectedNMHostNames));
+            hostsToRemove.entrySet().removeIf(e -> disconnectedNMHostNames.contains(e.getKey()));
```

Comment:
This check is case sensitive. please check.

Reply:
Converted all hostnames to lowercase and ensure lowercase is preserved during comparison